src/functional/ffront/fbuiltins.py

```diff
@@ -15,13 +15,22 @@
 from numpy import float32, float64, int32, int64
 
 from functional.common import Field
+from functional.ffront import common_types as ct
 
 
-__all__ = ["Field", "float32", "float64", "int32", "int64"]
+__all__ = ["Field", "float32", "float64", "int32", "int64", "nbh_sum"]
 
 TYPE_BUILTINS = [Field, float, float32, float64, int, int32, int64, bool, tuple]
 TYPE_BUILTIN_NAMES = [t.__name__ for t in TYPE_BUILTINS]
 
+nbh_sum = ct.FunctionType(
```

Comment:
`nbh_sum` is a bit too cryptic. I'd suggest to change it. Additionally, I now think it would be better to define the function as an actual (empty) function with the `__gt_type_` attribute set to the type definition. This change would be needed anyway for embedded execution and it feels like a cleaner design.